### PR TITLE
Fix(Core/ZulGurub): more improvements to High Priestess Jeklik

### PR DESCRIPTION
**Pull request by Nefertumm**
## Changes Proposed:
-  Move the scripts to the new system
-  Use the right spells for liquid fire.
-  Change her reset behaviour, she should tp to her balcony on wipe.
-  Add hover and gravity to bats.
-  Modify her run/walk speed, was way too high. (according to videos)

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11379
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11378
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11386 ? maybe

## SOURCE:

Walk/Run speed from CMangos, gob diameter from VMangos.

## Tests Performed:

- Tested in-game with 2 accounts.

## How to Test the Changes:

1. Engage with Jeklik and wipe.
2. Engage again and go to P2, wait for the bats to spawn.
3. :)

### CODE REVIEW

**Comment by Maelthyrr** [comment]
Tested and all work. Except one issue, she can summon 6 bats targetables in P2 like with "Summon bats" in P1.

**Comment by Nefertumm** [comment]
try again, please

File: src/server/scripts/EasternKingdoms/ZulGurub/boss_jeklik.cpp [hunk 1]
```diff
@@ -49,7 +52,8 @@ enum Spells
     SPELL_GREATER_HEAL          = 23954,
 
     // Batriders Spell
-    SPELL_BOMB                  = 40332 // Wrong ID but Magmadars bomb is not working...
+    SPELL_BOMB                  = 23970,
```
Comment (Kitzunu) [inline src/server/scripts/EasternKingdoms/ZulGurub/boss_jeklik.cpp:55]:
```suggestion
    SPELL_THROW_LIQUID_FIRE = 23970,
```

File: src/server/scripts/EasternKingdoms/ZulGurub/boss_jeklik.cpp [hunk 2]
```diff
@@ -49,7 +52,8 @@ enum Spells
...
+    SPELL_SPAWN_FLAMES          = 23971
```
Comment (Kitzunu) [inline src/server/scripts/EasternKingdoms/ZulGurub/boss_jeklik.cpp:56]:
```suggestion
    SPELL_SUMMON_LIQUID_FIRE = 23971
```

**Review by Kitzunu — CHANGES_REQUESTED** [review]

**Comment by Maelthyrr** [comment]
@Nefertumm I don't know if you have see, but you are some build errors with last update. I can't test for the moment.

**Comment by Nefertumm** [comment]
Oh yeah I'm kinda dumb, let me fix it real quick :P

**Review by Kitzunu — APPROVED** [review]

**Comment by Maelthyrr** [comment]
Everything is fine.

**Comment by Nyeriah** [comment]
build seems to be failing

**Pull request merged**